src/nerve/core/workflow/run.py

```
@@ -0,0 +1,392 @@
+
+    def _emit_event(self, event_type: str, data: dict[str, Any]) -> None:
+        """Emit event via callback (internal use)."""
+        if self._event_callback is None:
+            return
+
+        event = WorkflowEvent(
+            run_id=self._run_id,
+            workflow_id=self._workflow.id,
+            event_type=event_type,
+            data=data,
+            timestamp=datetime.now(),
```

Ruff (0.14.10)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)